aapcs64/aapcs64.rst

```diff
@@ -623,6 +672,46 @@ Homogeneous Short-Vector Aggregates (HVA)
 
 An Homogeneous Short-Vector Aggregate (HVA) is an Homogeneous Aggregate with a Fundamental Data Type that is a Short-Vector type and at most four uniquely addressable members.
 
+Pure Scalable Types (PSTs)
+--------------------------
+
+A type is a Pure Scalable Type if (recursively) it is:
+
+* a Scalable Vector Type;
+
+* a Scalable Predicate Type;
+
+* an array that contains a constant (nonzero) number of elements and whose
+  Base Type is a Pure Scalable Type; or
+
+* an aggregate in which every member is a Pure Scalable Type.
```

Comment:
I'd remove the semicolons and the or in this list. Looks cleaner.

```suggestion
* a Scalable Vector Type

* a Scalable Predicate Type

* an array that contains a constant (nonzero) number of elements and whose
  Base Type is a Pure Scalable Type

* an aggregate in which every member is a Pure Scalable Type
```

Reply:
I'd prefer to keep it as-is to be honest.  This is a relatively common style and makes the conjunction ("or") more obvious.  But maybe I've just been reading too many legal documents :-)

Reply:
sure, fine :)